okio-zipfilesystem/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+
+kotlin {
+  jvm {
+    withJava()
+  }
+  sourceSets {
+    commonMain {
+      dependencies {
+        api deps.kotlin.stdLib.common
+        api deps.kotlin.time
+        api project(":okio")
+      }
+    }
+    jvmTest {
+      dependencies {
+        implementation deps.test.junit
+        implementation deps.test.assertj
+        implementation deps.kotlin.test.jdk
+      }
+    }
+  }
+}
+
+// modify these lines for MANIFEST.MF properties or for specific bnd instructions
```

Comment:
Sentence

Reply:
Can do. Follow-up!

Reply:
The follow-up got rid of this comment! https://github.com/square/okio/pull/921/files#